src/commands/utils/wolframalpha.ts

```diff
@@ -1,82 +1,80 @@
 import { URLSearchParams } from 'url';
-import request from 'request';
+import fetch from 'node-fetch';
 import { CommandDefinition } from '../../lib/command';
 import { makeEmbed, makeLines } from '../../lib/embed';
-import { CommandCategory, Channels } from '../../constants';
+import { CommandCategory } from '../../constants';
 import Logger from '../../lib/logger';
 
-const WOFLRAMALPHA_API_URL = 'http://api.wolframalpha.com/v2/query?';
+const WOLFRAMALPHA_API_URL = 'http://api.wolframalpha.com/v2/query?';
 const WOLFRAMALPHA_QUERY_URL = 'http://www.wolframalpha.com/input/?';
 
 export const wolframalpha: CommandDefinition = {
     name: ['wa', 'calc', 'ask'],
     description: 'Queries the Wolfram Alpha API',
     category: CommandCategory.UTILS,
     executor: async (msg) => {
-        const allowedChannel = Channels.BOT_COMMANDS;
-        if (msg.channelId === allowedChannel) {
```

Comment:
Any reason that this was removed? We had it locked to #bot-commands to prevent spam

Reply:
Was just a decision made within mod team to have that one confined to #bot-commands